test/test_tracker.cpp

```diff
@@ -678,16 +680,29 @@ void test_stop_tracker_timeout(bool nostop)
 	char tracker_url[200];
 	std::snprintf(tracker_url, sizeof(tracker_url), "http://127.0.0.1:%d/announce", port);
 	announce_entry ae{tracker_url};
-	// trick to avoid use of tracker immediately
-	// FIXME
-	//ae.next_announce = aux::time_now32() + seconds32(1);
-	//ae.min_announce = aux::time_now32() + seconds32(1);
 	h.add_tracker(ae);
 
+	while (true)
+	{
+		std::vector<alert*> alerts;
+		s.pop_alerts(&alerts);
+		if (std::any_of(alerts.begin(), alerts.end()
+			, [](alert* a) { return a->type() == tracker_reply_alert::alert_type; }))
+			break;
+	}
+
 	s.remove_torrent(h);
 
 	int const count = count_stopped_events(s);
-	TEST_EQUAL(count, nostop ? 0 : 1);
+	if (nostop)
+	{
+		TEST_EQUAL(count, 0);
+	}
+	else
+	{
+		// the exact value will depend on the number of interfaces the host system has
```

Comment:
if the session is bound to a specific interface/IP, only that is used for the announce, right?